.github/workflows/CI.yml

```
@@ -64,28 +64,32 @@ jobs:
           name: CxbxReloaded-${{ matrix.configuration }}-${{ matrix.vsver }}
           path: artifacts
 
-  release: # TODO: Remove develop once rebased
-    if: github.event.action != 'pull_request' && github.ref == 'refs/heads/develop' || 'refs/heads/master' && github.repository == 'Cxbx-Reloaded/Cxbx-Reloaded'
+  release:
+    if: | # TODO: Remove develop once rebased
+      github.event.action != 'pull_request' &&
+      github.ref == 'refs/heads/develop'
+                 || 'refs/heads/master'     &&
+      github.repository == 'Cxbx-Reloaded/Cxbx-Reloaded'
```

Your multi-line commit didn't work as intended, I had verified this within my private Action-Test repo. Here's what you should change.
```suggestion
    if: | # TODO: Remove develop once rebased
      github.event.action != 'pull_request' &&
      (github.ref == 'refs/heads/develop' ||
       github.ref == 'refs/heads/master')     &&
      github.repository == 'Cxbx-Reloaded/Cxbx-Reloaded'
```

---

Otherwise, without above suggestion change. A new branch + add above code in a new commit will trigger job operation. If I add the change above, it will skip the job operation. Even skip it for a pull request to develop branch. Once merged, it will run the job. Again, make a pull request to master will not trigger it. Finally, merge pull request into master will trigger the job.